ui/seek_bar.js

```diff
@@ -623,10 +635,10 @@ shaka.ui.SeekBar = class extends shaka.ui.RangeElement {
     * Does a value compare on chapters.
     * @param {shaka.extern.Chapter} a
     * @param {shaka.extern.Chapter} b
-    * @return {!boolean}
+    * @return {boolean}
     */
     const chaptersEqual = (a, b) => {
-      return !(a && b) || (a.id === b.id && a.title === b.title &&
+      return !a || !b || (a.id === b.id && a.title === b.title &&
```

Comment:
Did you mean `(!a && !b)` as I said, or do you actually intend for `a == null` and `b != null` to be a match?

Reply:
I misunderstood the question, if it meant (!a && !b)